web/src/index.css

```diff
@@ -102,7 +102,20 @@
 
 input[type='datetime-local']::-webkit-datetime-edit-text { color: darkcyan; }
 
+@media (max-width: 576px) {
```

Comment:
Perhaps we can use the arbitrary value on Tailwind, so that we stay on Tailwind? 
https://tailwindcss.com/docs/responsive-design#arbitrary-values